modules/passkey/hardhat.config.ts

```diff
@@ -54,19 +66,19 @@ const config: HardhatUserConfig = {
     ...customNetwork,
   },
   solidity: {
-    compilers: [
-      {
-        version: '0.8.24',
+    compilers: [compilerSettings],
+    overrides: {
+      // FCL library does not optimize well via IR. In order to take advantage of the IR optimizer
+      // in the rest of the project without causing significant regressions to the FCL verifier, we
+      // add a compiler setting override for that specific contract.
+      'contracts/verifiers/FCLP256Verifier.sol': {
+        ...compilerSettings,
```

Comment:
```suggestion
        version: compilerSettings.version,
```

Reply:
I did this on purpose to future proof in case new fields were added. Agree it isn't strictly necessary here, but wanted to future proof just in case. Also it feels to convey intent a bit more IMO: we want exactly `compilerSettings` except for this single value.